extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/exporter/OtlpExporterTracesConfig.java

```diff
@@ -31,21 +31,6 @@ public interface OtlpExporterTracesConfig {
     @WithDefault(DEFAULT_GRPC_BASE_URI)
     Optional<String> legacyEndpoint();
 
-    //    /**
-    //     * Sets the certificate chain to use for verifying servers when TLS is enabled. The {@code byte[]}
-    //     * should contain an X.509 certificate collection in PEM format. If not set, TLS connections will
-    //     * use the system default trusted certificates.
-    //     */
-    //    @ConfigItem()
-    //    public Optional<byte[]> certificate;
```

Comment:
I know this is a pain but the certificate in pem format and the property names are normalised:
https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md

Example:
    config.put("otel.exporter.otlp.certificate", certificatePath);
    config.put("otel.exporter.otlp.client.key", clientKeyPath);

We might configure this in a more Quarkusy way but we need to document.

Reply:
I'd say let's do such improvements in a follow up

Reply:
For the time being, I followed what the rest of the Quarkus extensions do.

Reply:
Sounds good to me. Consistency in Quarkus is more important.

Reply:
That's been our approach so far (but no doubt we have made mistakes...)